README.md

```diff
@@ -17,6 +17,35 @@ To install BenchmarkTools, you can run the following:
 Pkg.add("BenchmarkTools")
 ```
 
+## Quick Start
```

Comment:
Can we move this section below the `Documentation` section?

Reply:
The whole point is that this is something people will read *before* looking at the full manual...

Reply:
The `Documentation` section is just a couple of links to the manual and reference guide, and it's quite normal style to include such links at the top of a README (along with test/CI badges, etc). This way, users can immediately see what documentation is available without having to dig through other documentation to get there, and it doesn't negatively impact the usefulness of the quick start guide in any way.

Reply:
Okay.